pkg/appmgmt/interfaces/recoverable.go

```diff
@@ -35,7 +35,7 @@ import (
 // and then properly recover these applications before recovering nodes.
 type Recoverable interface {
 	// list applications returns all existing applications known to this app manager.
-	ListApplications() (map[string]ApplicationMetadata, error)
+	ListApplications() ([]*v1.Pod, error)
```

Comment:
Let's call this ListPods() or something since apps don't have anything to do with it any more.

Reply:
Done